documentation/general/dash-sonic-hld.md

```diff
@@ -311,8 +321,10 @@ DASH_ACL_RULE_TABLE:{{group_id}}:{{rule_num}}
     "action": {{action}}
     "terminating": {{bool}}
     "protocol": {{list of protocols}}
-    "src_addr": {{list of address}}
-    "dst_addr": {{list of address}}
+    "src_tag": {{list of tag name}}
```

Comment:
Mark this optional

Reply:
Should we have an explicit "optional" mark to "tag" field? IMHO, most of fields in DASH ACL RULE TABLE are optional. E.G. We can only set src_tag or dst_port and set other fields to be empty. So, should we also mark them?